src/msg.rs

```diff
@@ -5,28 +5,28 @@ use serde::{Deserialize, Serialize};
 
 use crate::batch;
 use crate::transaction_history::{RichTx, Tx};
-use crate::viewing_key::ViewingKey;
-use cosmwasm_std::{Binary, HumanAddr, StdError, StdResult, Uint128};
+use crate::viewing_key_obj::ViewingKeyObj;
+use cosmwasm_std::{Addr, Binary, StdError, StdResult, Uint128};
 use secret_toolkit::permit::Permit;
 
-#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema)]
 pub struct InitialBalance {
-    pub address: HumanAddr,
+    pub address: Addr,
     pub amount: Uint128,
 }
 
 #[derive(Serialize, Deserialize, JsonSchema)]
-pub struct InitMsg {
+pub struct InstantiateMsg {
     pub name: String,
-    pub admin: Option<HumanAddr>,
+    pub admin: Option<String>,
```

Comment:
Because Addr (or HumanAddr in V10) represent a validated address.
This is something that won't necessarily occur because the user can send a wrong address. 

So for the input messages - we get a String and then verifies it's indeed an address.
For the output messages - we send Addr because an address that comes from the contract is validated as we consider our contract to be a trusted & validated source.